CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -739,73 +679,16 @@ private int UpdateUniformBuffers(MemoryStack stack) {
         final Matrix44f translationMatrix = new Matrix44f();
         translationMatrix.makeTranslationMatrix(topRightCorner.getX(), 
                                                 topRightCorner.getY(), 
-                                                topRightCorner.getZ()); 
-        vertexUBO.mvpMatrix.multiply(translationMatrix, srMatrix); 
+                                                topRightCorner.getZ());       
         
-        // Populate the staging buffer
-        int size = VertexUniformBufferObject.SIZEOF;
-        PointerBuffer vertData = stack.mallocPointer(1);
-        vkMapMemory(cvkDevice.GetDevice(), cvkStagingBuffer.GetMemoryBufferHandle(), 0, size, 0, vertData);
-        {
-            vertexUBO.CopyTo(vertData.getByteBuffer(0, size));
-        }
-        vkUnmapMemory(cvkDevice.GetDevice(), cvkStagingBuffer.GetMemoryBufferHandle());        
-          
-        for (int i = 0; i < imageCount; ++i) {   
-            CVKBuffer vertUniformBuffer = vertexUniformBuffers.get(i);
-            vertUniformBuffer.CopyFrom(cvkStagingBuffer);
-        }
+        // Calculate the model-view-projection matrix
+        vertexUBO.mvpMatrix.multiply(translationMatrix, srMatrix);
+        
+        // Update the push constants data
+        pushConstants.clear();
```

Comment:
Is the clear needed if we are blatting right on top of it?

Reply:
True not needed as we do a flip() after, removed

Reply:
Do we even need the flip?  From memory that moves the position back to 0.  CopyTo may do that already.

Reply:
Yes definitely needed. Otherwise the position is at the end and it will fail on the CmdPushConstant()